minio/api.py

```diff
@@ -698,6 +698,8 @@ def fget_object(self, bucket_name, object_name, file_path):
         :param bucket_name: Bucket to read object from.
         :param object_name: Name of the object to read.
         :param file_path: Local file path to save the object.
+        :param metadata: Any additional metadata to be uploaded along
```

Comment:
Are you sure this is what you want to do? GET doesn't upload any object in-fact it downloads the object. How does adding another parameter to fget_object help here? .